mbtools/validate_mbz_html.py

```diff
@@ -66,6 +70,35 @@ def validate_mbz(mbz_path, include_styles=True, include_questionbank=False):
     return violations
 
 
+def validate_html(html_dir, include_styles=True):
+    all_files = []
+    for path, dirs, files in os.walk(html_dir):
+        for file in files:
+            all_files.append(os.path.join(path, file))
+    html_elements = []
+    for file_path in all_files:
+        with open(file_path, 'r') as f:
+            parent_string = '<content></content>'
+            parent_element = html.fragments_fromstring(parent_string)[0]
+            print(file_path)
```

Comment:
Was this `print` from debugging? It might be a bit verbose for logging in practice.